src/clockface/components/context_menu/ContextMenu.tsx

```diff
@@ -60,9 +63,10 @@ class ContextMenu extends Component<Props, State> {
             text={text}
             shape={shape}
             icon={icon}
-            size={ComponentSize.ExtraSmall}
+            size={size}
             color={color}
             testID={testID}
+            style={style}
```

Comment:
Is there a reason why we're creating a new styling pattern for this specific page?

Reply:
I wanted to add margin to the Context.Menu for the delete button inside the SecretContextMenu.  It seemed simplest and easiest to allow an optional style to be passed in and used as needed for the purpose.  I'm open to other alternatives though.

Reply:
No problem with the implementation, I just wonder if we wanted to introduce a new design pattern for the app based on design asks or not